.travis.yml

```diff
@@ -20,41 +21,17 @@ jobs:
       script:
         - pre-commit autoupdate
         - pre-commit run --all-files
-        - sam build --build-dir lambdas/build/jumpcloud --base-dir lambdas/jumpcloud --template lambdas/jumpcloud/template.yaml
-        - sam build --build-dir lambdas/build/rotate-credentials --base-dir lambdas/rotate-credentials --template lambdas/rotate-credentials/template.yaml
-        - sam build --build-dir lambdas/build/cfn-explode-macro --base-dir lambdas/cfn-explode-macro --template lambdas/cfn-explode-macro/template.yaml
-        - sam build --build-dir lambdas/build/cfn-ssm-param-macro --base-dir lambdas/cfn-ssm-param-macro --template lambdas/cfn-ssm-param-macro/template.yaml
-        - sam build --build-dir lambdas/build/cfn-s3objects-macro --base-dir lambdas/cfn-s3objects-macro --template lambdas/cfn-s3objects-macro/template.yaml
-        - sam build --build-dir lambdas/build/cfn-oidc-identity-provider --base-dir lambdas/cfn-oidc-identity-provider --template lambdas/cfn-oidc-identity-provider/template.yaml
+        - ./build-lambdas.sh
```

Comment:
if your trying to change the travis build then i would prefer this to happen in a separate PR

Reply:
Ok, I'll move this.